package.json

```diff
@@ -8,7 +8,8 @@
     "bundle": "webpack --config webpack.prod.js && npm run tsc",
     "docs": "jsdoc -c jsdoc.json",
     "tsc": "tsc --project lib/tsconfig.json && tscpaths -p lib/tsconfig.json -s lib -o dist/@types",
-    "test": "node test-server.js"
+    "test": "node test-server.js",
+    "lint": "eslint . --ext .ts"
```

Comment:
We can just have `eslint` here and in `.eslintrc` set the `overrides` option like so:

```json
...
"overrides": [
    {
      "files": ["lib/**/*.ts"]
    }
]
...
```

This will lint `*.ts` files inside the `lib` directory only.

Reply:
@johnny243 have you tried this option?

I've tried `eslint .` and `eslint lib` with: 
- `"files": ["lib/**/*.ts"]`
- `"files": ["**/*.ts"]`
- `"files": ["*.ts"]`

none of those work as expected (pick up only js files in the root)